### PR TITLE
Try postgres

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/MunicipalityRegistry.Projections.Integration/MunicipalityLatestItem.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    {
+        public void Configure(EntityTypeBuilder<MunicipalityLatestItem> builder)
+        {
+            builder
+                .ToTable("municipality_latest_items", Schema.Integration) // to schema per type
+                .HasKey(x => x.MunicipalityId);
+
+            builder.Property(x => x.MunicipalityId).HasColumnName("municipality_id");
+            builder.Property(x => x.NisCode).HasColumnName("nis_code");
+            builder.Property(x => x.Status).HasColumnName("status");
```
Comment (jvandaal) [inline src/MunicipalityRegistry.Projections.Integration/MunicipalityLatestItem.cs:67]: Should we limit the column size of Status (or other fields)?

File: src/MunicipalityRegistry.Projections.Integration/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyDescription("MunicipalityRegistry Legacy Projections")]
```
Comment (jvandaal) [inline src/MunicipalityRegistry.Projections.Integration/Properties/AssemblyInfo.cs:4]: Should be Integration Projections

File: src/MunicipalityRegistry.Projections.Integration/MunicipalityVersionExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    public static class MunicipalityVersionExtensions
+    {
+        public static async Task CreateNewMunicipalityVersion<T>(
+            this IntegrationContext context,
+            Guid municipalityId,
+            Envelope<T> message,
+            Action<MunicipalityVersion> applyEventInfoOn,
+            CancellationToken ct) where T : IHasProvenance, IMessage
+        {
+            var municipalitySyndicationItem = await context.LatestPosition(municipalityId, ct);
```
Comment (jvandaal) [inline src/MunicipalityRegistry.Projections.Integration/MunicipalityVersionExtensions.cs:22]: Rename municipalitySyndicationItem to municipalityVersion

File: src/EF.MigrationsHelper/appsettings.json [hunk 4]
```diff
@@ -5,7 +5,8 @@
     "LastChangedListAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.MunicipalityRegistry;Trusted_Connection=True;",
 	  "ProducerProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.MunicipalityRegistry;Trusted_Connection=True;",
     "WfsProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.MunicipalityRegistry;Trusted_Connection=True;",
-    "WmsProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.MunicipalityRegistry;Trusted_Connection=True;"
+    "WmsProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.MunicipalityRegistry;Trusted_Connection=True;",
+    "IntegrationProjectionsAdmin": "Host=vbr-integrationdb-test.postgres.database.azure.com;Database=postgres;Username=basisregisters;Password=Rj7Hnudk02zie4eA2N2jkSeIAEflbQe7e6DwN7Dhe9gDAFR7akvfvLVx2KOXo5Pkjd3BFiALyUkcbjoxI29R1zGT3dWhdJDhu0A"
```
Comment (jvandaal) [inline src/EF.MigrationsHelper/appsettings.json:9]: Leaked password?

File: src/MunicipalityRegistry.Projector/Infrastructure/Modules/ApiModule.cs [hunk 5]
```diff
@@ -134,6 +138,23 @@ private void RegisterLegacyProjections(ContainerBuilder builder)
...
+                    new IntegrationModule(
+                        _configuration,
+                        _services,
+                        _loggerFactory));
+
+            builder
+                .RegisterProjectionMigrator<IntegrationContextMigrationFactory>(
+                    _configuration,
+                    _loggerFactory)
+                .RegisterProjections<MunicipalityLatestItemProjections, IntegrationContext>(
```
Comment (jvandaal) [inline src/MunicipalityRegistry.Projector/Infrastructure/Modules/ApiModule.cs:154]: We should also register the MunicipalityVersion projections

**Review by ArneD — APPROVED** [review]

**Comment by ArneD** [comment]
Can you add a toggle in the projector? `EnableIntegration`

**Pull request merged**